app/controllers/map_controller.rb

```diff
@@ -16,7 +16,10 @@ def index
   end
 
   def show
-    @node = DrupalNode.find_map_by_slug(params[:name]+'/'+params[:date])
+    @node = DrupalNode.find params[:name]
```

Comment:
I think @ebarry and @steviepubliclab may join me in wanting to avoid the need for `--2` to be added to URLs if possible. 

If the `find` method won't accept a '/' character (why won't it, is there a validation?) could we do:

```js
  @node = DrupalNode.find params[:name] + '#' + params[:date]
```

We'll surely then have to override the *generation* of the friendly_id similarly to substitute a '#' or another suitable character. But in any case, maps use a different route, so we can add logic of this kind before using the `find` method, or before creating the unique id for friendly_id's use. 

I know this is not ideal, but it is not also not ideal to change our map URLs (even if we do redirect). I also want to note that it's pretty typical of any web project to be extremely resistant to URL schema changes, especially if the reason is simply that a library doesn't support a schema by default. 

Do you see a problem with this (or a similar) approach, and would you be willing to give it a try? Otherwise I think you'll have to make a strong case to convince everyone that a URL change is worthwhile, for the benefits in this PR. 

Thanks.